### ISSUE TITLE
[NIGHTWACH E2E]: Public Websites - axeCheck must be updated in Site / Tests / Home / 00 Required E2e Spec or test converted to Cypress

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21805]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe false positive violations were detected for color contrast. The best course of action now is to either update the `axeCheck()` call to use a custom config to ignore the color contrast rule (in progress) or a better approach will be to convert this test to Cypress.

## Related Issue:
* [[AUTOMATION]: Update Nightwatch axeCheckJS to capture failing tests ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/21738)

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations
- [ ] Nightwatch e2e test `00 Required E2e Spec` passes axeCheck validation OR
- [ ] Test is converted to Cypress and passes axeCheck validation

## Environment
* src/site/tests/home/00-required.e2e.spec.js

**Comment by brianalloyd** [comment]
Can you confirm these are still issues, might have had duplicate issues/tickets that we worked. @1Copenut TIA
